Templates/BlobTrigger-PowerShell/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "BlobTrigger - PowerShell",
+    "description": "A PowerShell function that will be run whenever a blob is added to a specified container",
+    "defaultFunctionName": "BlobTriggerPowerShell",
+    "language": "PowerShell",
+    "category": [

```

Comment:
Currently all our Powershell templates are in the "Experimental" category. We should be consistent here. Once this new model is solidified, we may decide to promote Powershell, but we'll do that in one go, across all the templates.